Header header

int32 id
int32[] neighbour_ids
geometry_msgs/Point room_length
geometry_msgs/Point room_center
geometry_msgs/Point cluster_center
PlaneData[] x_planes
PlaneData[] y_planes